main.go

```diff
@@ -1712,6 +1713,16 @@ func main() {
 		ocdCommands = strings.Split(*ocdCommandsString, ",")
 	}
 
+	val, ok := os.LookupEnv("TINYGOGOCOMPATIBILITY")
```

Comment:
Might be better as `TINYGO_GOCOMPATIBILITY` to make it more readable?

Reply:
Thanks, I addressed it.